index.js

```diff
@@ -84,7 +84,7 @@ const projectLists = [
 ];
 
 const togglePopUp = (e) => {
-  const itemId = e.originalTarget.id;
+  const itemId = e.target.id;
   projectLists.find((item) => {
     let hi;
```

Comment:
### Your Modal Window Mobile
![image](https://user-images.githubusercontent.com/93039359/195405609-44b2ab6f-3d65-4e96-b368-665ed8c89721.png)
### Your Modal WIndow Desktop
![image](https://user-images.githubusercontent.com/93039359/195406633-5955a3cf-5fd9-4ac6-98bf-c662be9e0cf9.png)


- [ ] You can see that you modal window on mobile 375px screen size. You can see from top it going out of your window from top and it is not scrollable. Kindly fix your design from top :blush: to make it look professional and